packages/nimble-tokens/source/styledictionary/build.js

```diff
@@ -0,0 +1,100 @@
+require('./config');
+require('./nimble-extensions');
+const StyleDictionary = require('style-dictionary');
+
```

Comment:
@jattasNI pushed a change that makes this workflow a bit more normal. These imports register new formatters with StyleDictionary shared global and this file actually defines the config and runs the builds with those registered formatters.